commands.js

```diff
@@ -66,19 +66,43 @@ addUser = async (user, message) => {
     return info;
 }
 
+/**
+ * Adds a transaction to an existing user
+ * Params: userSchema user, object message
+ * Returns model of transaction info that was updated to the user
+*/
 addTransactions = async (user, message) => {
 
+    let transactionInfo = new TransactionSchema();
+    transactionInfo.name = message.text.name;
+    transactionInfo.amount = message.text.amount;
+    transactionInfo.other = message.text.other;
+    transactionInfo.date = message.text.date;
+    transactionInfo.details = message.text.details;
+
     try {
+        const doc = await User.find({
+            username: user.username
+        });
+
+        if (!doc) {
+            reply = `You must register yourself with the bot first, try: '/start'`
```

Comment:
Please put the string 
```You must register yourself with the bot first...```
In strings.js, import that file (as customStrings) and use customStrings.variableName to refer to that string.

Reply:
sure thing, I'll update the code and pull the string text from there.